### PR TITLE
add button from UI Kit

### PR DESCRIPTION
**Pull request by Le-Caignec**

### CODE REVIEW

File: src/features/newProtectedData/NewProtectedData.tsx
Comment (PierreJeanjacquot) [inline src/features/newProtectedData/NewProtectedData.tsx:1]: not from this PR but I noticed a bug, `name` is required to create a protected data in `handleSubmit`, it should not

```ts
  const handleSubmit = async () => {
    const data: any = {};
    let bufferFile: ArrayBuffer;
    switch (dataType) {
      case 'email':
        data['email'] = email;
        break;
      case 'file':
        bufferFile = await createArrayBufferFromFile(file);
        data['file'] = bufferFile;
        break;
    }
    if (dataType && name && ((isValidEmail && email) || file)) { // <= here
      await createProtectedData({ data, name });
    }
  };
```

File: src/features/newProtectedData/NewProtectedData.tsx [hunk 1]
```diff
@@ -111,15 +112,17 @@ export default function NewProtectedData() {
...
           {!filePath ? 'Upload' : 'Updated File'}
-          <input hidden multiple type="file" />
+          <input
+            ref={fileInput}
+            hidden
+            multiple
+            type="file"
+            onChange={(e) => handleFileChange(e)}
+          />
         </Button>
```
Comment (PierreJeanjacquot) [inline src/features/newProtectedData/NewProtectedData.tsx:126]: the upload button looks a bit aggressive maybe we should use the 'secondary' variant

| current | proposed |
| --- | --- |
| image | image |

**Review by PierreJeanjacquot — CHANGES_REQUESTED** [review]
- old bug spotted
- UI suggestion

File: src/features/newProtectedData/NewProtectedData.tsx
Comment (Le-Caignec) [reply src/features/newProtectedData/NewProtectedData.tsx:1]: i create a Ticket in the backlog https://iexecproject.atlassian.net/browse/PRO-327?atlOrigin=eyJpIjoiMGVhNzBlZTE3YmJlNGMxMzhhYjM5OWFkYjhjNWIxM2MiLCJwIjoiaiJ9

File: src/features/newProtectedData/NewProtectedData.tsx (hunk 1 above)
Comment (Le-Caignec) [reply src/features/newProtectedData/NewProtectedData.tsx]: Commit: 343ef19e4dbccc3615f72e826e8f06e5ef71d4c6

Comment (Le-Caignec) [reply src/features/newProtectedData/NewProtectedData.tsx]: goog idear

**Review by PierreJeanjacquot — APPROVED** [review]
LGTM

**Pull request merged**